website/docs/documentation/Pipelines/Smart-seq2_Single_Nucleus_Pipeline/README.md

```diff
@@ -0,0 +1,136 @@
+| [Picard.CollectMultipleMetrics](https://github.com/broadinstitute/warp/tree/develop/tasks/skylab/Picard.wdl) | Collects QC metrics on the deduplicated BAM file | Picard | CollectMultipleMetrics |
+| [CountAlignments.CountAlignments](https://github.com/broadinstitute/warp/tree/develop/tasks/skylab/FeatureCounts.wdl) | Uses a custom GTF with featureCounts and Python to mark introns, create a BAM that has alignments spanning intron-exon junctions removed, and counts exons using the custom BAM and by excluding intron tags | [Subread](http://subread.sourceforge.net/) | [FeatureCounts](http://bioinf.wehi.edu.au/featureCounts/), Python 3 | 
+| [LoomUtils.SingleNucleiSmartSeq2LoomOutput](https://github.com/broadinstitute/warp/tree/develop/tasks/skylab/LoomUtils.wdl) | Combines featureCounts and Picard metric outputs into a Loom formatted cell-by-gene matrix | Python 3 | [Custom script](https://github.com/broadinstitute/warp/tree/develop/dockers/skylab/loom-output/create_loom_snss2.py)
+
+### Trimming Adaptors
+The TrimAdaptors task uses the adaptor list reference file to run the [fastq-mcf](https://github.com/ExpressionAnalysis/ea-utils/tree/master/clipper) tool. This tool identifies the adaptors in the input FASTQ files and performs clipping by using a subsampling parameter of 200,000 reads. The task outputs the trimmed FASTQ files which are then used for alignment. 
+
+### Aligning Reads
+The StarAlignFastq task runs the STAR aligner on the trimmed FASTQ files. The STAR quantMode parameter is set to GeneCounts, which counts the number of reads per gene while mapping. The task outputs a coordinate-sorted aligned BAM file. 
+
+### Removing Duplicate Reads 
+The RemoveDuplicatesFromBam task removes multi-mapped reads, optical duplicates, and PCR duplicates from the aligned BAM. It then adds readgroup information and creates a new, coordinate-sorted aligned BAM output.
+
+### Collecting Metrics
+The CollectMultipleMetrics task uses the Picard tool CollectMultipleMetrics to perform QC on the deduplicated BAM file. These metrics are copied to the final cell-by-gene matrix output (Loom file).
+
+### Counting Genes
+The CountAlignments task uses the featureCounts package to count introns and exons. First, the featureCounts tool counts intronic alignments in the deduplicated BAM using a custom GTF with annotated introns. The tool flags intronic alignments if they overlap an annotated intron by a minimum of 3 bp. 
+
+Next, following pipeline processes established by the BICCN Whole Mouse Brain Working Group, a custom Python script (“[remove-reads-on-junctions.py](https://github.com/broadinstitute/warp/tree/develop/dockers/skylab/featureCounts/remove-reads-on-junctions.py)”) removes all alignments in the deduplicated BAM that cross only one intron-exon junction and produces an intermediate BAM file for exon counting. This removes a negligible amount of putative intronic alignments that did not meet the 3 bp intron overlap criteria. 
```

Comment:
We don't output this intermediate bam, right?

Reply:
No, it's not a pipeline output.

Reply:
Thanks!

Reply:
I think the way the code is written, we remove any reads spanning a partial intron regardless of how much intron overlap they have. This is done after counting introns, but before counting exons